click_odoo_contrib/initdb.py

```diff
@@ -64,7 +64,14 @@ def odoo_createdb(dbname, demo, module_names, force_db_storage):
         odoo.service.db._create_empty_database(dbname)
         odoo.tools.config["init"] = dict.fromkeys(module_names, 1)
         odoo.tools.config["without_demo"] = not demo
-        odoo.modules.registry.Registry.new(dbname, force_demo=demo, update_module=True)
+        if odoo.release.version_info < (19, 0):
+            odoo.modules.registry.Registry.new(
+                dbname, force_demo=demo, update_module=True
+            )
+        else:
+            odoo.modules.registry.Registry.new(
+                dbname, new_db_demo=demo, update_module=True
+            )
```

Comment:
I don't know what is the most readable condition
```suggestion
        if odoo.release.version_info >= (19, 0):
            odoo.modules.registry.Registry.new(
                dbname, new_db_demo=demo, update_module=True
            )
            
        else:
            odoo.modules.registry.Registry.new(
                dbname, force_demo=demo, update_module=True
            )
```